python/grass/tools/tests/grass_tools_session_tools_test.py

```diff
@@ -0,0 +1,812 @@
+"""Test grass.tools.Tools class"""
+
+import io
+import os
+
+import pytest
+
+import grass.script as gs
+from grass.experimental.mapset import TemporaryMapsetSession
+from grass.tools import Tools
+from grass.exceptions import CalledModuleError
```

Comment:
isort+formatting gives:

```suggestion
import io
import os

import pytest

import grass.script as gs
from grass.exceptions import CalledModuleError
from grass.experimental.mapset import TemporaryMapsetSession
from grass.tools import Tools
```